api/database/migrations/003_sar_side_car_files.down.sql

```diff
@@ -0,0 +1,4 @@
+
+ALTER TABLE media
+  DROP COLUMN IF EXISTS side_car_path,
+  DrOP COLUMN IF EXISTS side_car_hash;
```

Comment:
Small typo in DROP

Reply:
Corrected and I also corrected it in 002_site_info_more_settings.down.sql :)